### PR TITLE
Implement timecard UX for displaying worker time entries

### PR DESCRIPTION
**Pull request by jrbotros**
Also removed view logic from the `TimeEntry` prototype.

### CODE REVIEW

File: orchestra/static/orchestra/common/js/orchestra.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
   'use strict';
 
   var modules = ['orchestra.routes', 'orchestra.config', 'orchestra.dashboard',
-    'orchestra.task', 'orchestra.project_management', 'orchestra.common'
+    'orchestra.task', 'orchestra.timing', 'orchestra.project_management', 'orchestra.common',
```
Comment (marcua) [inline orchestra/static/orchestra/common/js/orchestra.js]: trailing , ok?

File: orchestra/static/orchestra/timing/timecard/controller.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      vm.timeEntries = timeEntries;
+
+      vm.datetimeFromKey = function(entries) {
+        if (entries.$key) {
+          return moment(entries.$key);
+        }
+      };
+
+      vm.humanizeDuration = function(duration) {
+        return duration.get('hours') + 'h ' + duration.get('minutes') + 'm';
```
Comment (marcua) [inline orchestra/static/orchestra/timing/timecard/controller.js:18]: this appears to truncate, rather than round up.

can we round up consistently throughout?

File: orchestra/static/orchestra/common/js/orchestra.js (hunk 1 above)
Comment (jrbotros) [inline orchestra/static/orchestra/common/js/orchestra.js]: Technically fine but not necessary—also going to reorganize these.

File: orchestra/static/orchestra/timing/timecard/controller.js (hunk 2 above)
Comment (jrbotros) [inline orchestra/static/orchestra/timing/timecard/controller.js:18]: Yes, let me do that in a separate PR.

**Comment by marcua** [comment]
lgtm
